### PR TITLE
fix(Core/Script): Felmyst

### PR DESCRIPTION
**Pull request by h0lysp4nk**
First of all big thanks to @BarbzYHOOL for the SQL, support and advice on changes that needed to be made!

## CHANGES PROPOSED:
- Corrected dialog text in `creature_text` table
- Corrected translations
- Added another line to Remove the charm Aura from the player.

## ISSUES ADDRESSED:
Felmyst (boss in Sunwell Plateau) has these issues:

- If she resets while you're mind controlled, you fly like an idiot (can be the cause of the bug below)
- If you get kicked during her fight when she mind controls you, when you log back, your player has a new aura and he's incredibly strong because the aura couldn't get remove in time
- When she resets, she becomes passive, and so she doesn't aggro you in range (trivial issue)
- When she attacks you with her Mind Control spell (fog of corruption), she says a line and she says the WRONG line
- When she dies, kalecgos says the wrong line
- She's missing a line when she attacks

## TESTS PERFORMED:
- Built the core
- Tested in-game

## HOW TO TEST THE CHANGES:

- Build the core using both mine and @BarbzYHOOL changes
- .tele sunwell
- Fight Felmyst

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1588269002996885593.sql [hunk 1]
```diff
@@ -0,0 +1,46 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1588269002996885593');
+
+-- Fix Kalec msg on Felmyst death
+UPDATE `creature_text` SET
+`Text` = 'Madrigosa deserved a far better fate. You did what had to be done, but this battle is far from over!',
+`Sound` = '12439',
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588269002996885593.sql:6]: Don't have `''` around numbers. pls update the entire sql

**Comment by merdlin** [comment]
Waiting for the change to be tested.

File: docker/database/Dockerfile [hunk 2]
```diff
@@ -24,3 +24,5 @@ COPY --from=builder /azerothcore/env/dist/sql /sql
 # adding the "generate-databases.sh" to the directory "/docker-entrypoint-initdb.d"
 # because all scripts included in that directory will automatically be executed when the docker container starts
 COPY docker/database/generate-databases.sh /docker-entrypoint-initdb.d
+
+HEALTHCHECK --interval=5s --timeout=15s --start-period=30s --retries=3 CMD mysqladmin ping -h localhost
```
Comment (P-Kito) [inline docker/database/Dockerfile:28]: Is that ninjaed in here? Those docker changes

Comment (h0lysp4nk) [reply docker/database/Dockerfile]: Sorry about that. The docker changes were from another PR that I used as a base. I've removed them from this PR.

**Comment by h0lysp4nk** [comment]
> Pls remove the docker stuff since they have nothing to do with this PR

Done, sorry. They were from another PR that I used as a base.

**Comment by merdlin** [comment]
Tested, it works.

**Pull request merged**